docs/pages/build-reference/local-builds.mdx

```diff
@@ -5,14 +5,14 @@ description: Learn how to use EAS Build locally on your machine or a custom infr
 
 import { Terminal } from '~/ui/components/Snippet';
 
-You can run the same build process that we run on the EAS Build servers directly on your machine with the `--local` flag.
+You can run the same build process that we run on the EAS Build servers directly on your machine with the `--local` flag which is a useful way to debug build failures happening on your EAS cloud builds.
+
+> **Note:** To make builds using the Expo framework locally use [`npx expo run:android` or `npx expo run:ios` commands](/more/expo-cli/#compiling) instead. The `eas build --local` command is a way to make builds using [EAS cloud services](/build/introduction) locally. If you use [Continuous Native Generation](/workflow/continuous-native-generation), you can also run [prebuild](/workflow/prebuild) to generate your **android** and **ios** directories and then proceed to open the projects in the respective IDEs. Further, you can build them just like any native project.
```

Comment:
```suggestion
> **Note:** The `eas build --local` command is a way to make builds using [EAS cloud services](/build/introduction) on your local machine, following the same process that would otherwise run remotely on our cloud. To compile and run your app locally with Expo CLI, use [`npx expo run:android` or `npx expo run:ios` commands](/more/expo-cli/#compiling) instead.  If you use [Continuous Native Generation](/workflow/continuous-native-generation), you can also run [prebuild](/workflow/prebuild) to generate your **android** and **ios** directories and then proceed to open the projects in the respective IDEs. Further, you can build them just like any native project.
```